tsconfig.json

```diff
@@ -7,7 +7,7 @@
         "rootDir": "src",
         "strict": true,
         "strictPropertyInitialization": false,
-        "target": "es2017",
+        "target": "es2022",
```

Comment:
I have no idea if what i'm asking there is stupid tbh ^^

But should we update the rest of these options, like 

```
"lib": ["dom", "es2015"],
"module": "es2015",
```

Reply:
`target` already modify `lib `, but yeah we can